tests/expect/test_fish_completion.exp

```diff
@@ -16,17 +16,11 @@ set len [expr [string length $in_line] - 1]
 set line [string range $in_line 5 $len]
 set num_tabs [lindex $argv 2]
 puts "input line: '$line'"
-puts "num_tabs: $num_tabs"
 
 send "[lindex $argv 0] -sc install=fish | source\r"
 expect >
 
-set command "[lindex $argv 0] $line"
-while {$num_tabs > 0} {
-    incr num_tabs -1
-    append command "\t"
-}
-
```

Comment:
this makes sense to me.
the whole thing with multiple tabs is very bash specific.